managed/services/telemetry/config.default.yml

```diff
@@ -57,6 +57,14 @@ telemetry:
       - metric_name: "mysql_version"
         label: "version"
 
+- id: MySQLVersionComment
+    source: VM
+    query: mysql_version_info{service_type="mysql"}
+    summary: "Detailed information about version of monitored MySQL services"
+    data:
+      - metric_name: "mysql_version"
+        label: "version_comment"
```

Comment:
@ritbl can we include this data in the block above?

Reply:
It was my typo. this need to be a new metric name